.github/workflows/macos-release.yml

```
@@ -0,0 +1,258 @@
+              echo "Found latest-mac.yml: $LATEST_YML"
+          else
+              echo "⚠️ WARNING: latest-mac.yml not found"
+          fi
+          
+          # Create final filename
+          FINAL_DMG_NAME="giddh-${PACKAGE_VERSION}.dmg"
+          echo "FINAL_DMG_NAME=$FINAL_DMG_NAME" >> $GITHUB_ENV
+          echo "Final DMG name will be: $FINAL_DMG_NAME"
+
+      - name: Configure AWS credentials (OIDC)
+        if: ${{ secrets.AWS_ROLE_TO_ASSUME != '' }}
```

actionlint (1.7.10)

context "secrets" is not allowed here. available contexts are "env", "github", "inputs", "job", "matrix", "needs", "runner", "steps", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)